packages/flutter/test/material/app_bar_theme_test.dart

```diff
@@ -0,0 +1,106 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('AppBar theme does not affect defaults', (WidgetTester tester) async {
```

Comment:
This is more like 'No AppBar theme returns defaults.'

Reply:
Updated